ts/model.ts

```diff
@@ -541,7 +541,8 @@ abstract class NohmModel<TProps extends IDictionary = IDictionary> {
   ): Array<void | IPropertyDiff<keyof TProps>> {
```

Comment:
The void return types can then be removed.

Reply:
updated in the PR